src/kili/queries/data_connection/__init__.py

```diff
@@ -118,3 +119,32 @@ def cloud_storage_connections(
         if as_generator:
             return data_connections_gen
         return list(data_connections_gen)
+
+    @typechecked
+    def cloud_storage_connection(
```

Comment:
no unfortunately the dataconnections query is made like this:

```graphql
  dataConnections(where: DataConnectionsWhere!, first: PageSize!, skip: Int!): [DataConnection!]
input DataConnectionsWhere {
  projectId: ID
  integrationId: ID
}
```

The docstring is clear enough for cloud_storage_connection() no? "Get information of a cloud storage connection."

If you prefer, I can hide everything under cloud_storage_connections():
- if cloud_storage_connection_id is provided, I call the resolver dataConnection
- if not, I call dataConnections

Reply:
I admit that I would have implemented it that way, with only one entrypoint. Because all other queries are implemented in the same way and that at the end they return the same result isn't it? Both can return the same fields. It is just a matter of returning one unique dict or a list of dict?
If so, I would personally have one function `cloud_storage_connections` for the moment, like this we don't take any risk and don't break the old pattern. And if we feel that it is something to change, we can duplicate it. It can be discussed tomorrow in the retro. 
@PierreLeveau an opinion ?

Reply:
yeah I'm good with that! Better stay consistent with existing methods